sdk/include/xdk/interlocked.h

```diff
@@ -173,6 +173,12 @@
 /* No release here */
 #define InterlockedExchangeNoFence64 __NF_(_InterlockedExchange64)
 
+#ifdef _WIN64
+#define InterlockedExchangeSizeT(a, b) InterlockedExchange64((LONG64 *)a, b)
```

Comment:
Microsoft's headers doesn't seem to have this one, so we shouldn't add it either. Could just use `InterlockedExchangeAddSizeT` instead I suppose.

Reply:
This macro is just to exchange between two values and return that , not to perform an addition operation -- which is used in `PspReturnExcessQuotas`. Unless you came up with an idea of using `InterlockedExchangeAddSizeT` but in a tricky way for that or what exactly?

Reply:
Yep, the tricky way is to add 0. `InterlockedExchangeAdd(&var, 0)` is the same as `InterlockedExchange(&var)`